### DISCUSSION TITLE
Batch Analyses

### DISCUSSION THREAD

**Discussion by nshobert** [MoiseRousseau/PyGeoStudio#18] [General]
Hi Moise and other users,

Thanks for sharing your work on this, it looks like a lot of effort. I'm looking for a way to batch analyses, running each item in a batch varying only one parameter at a time. It surprised that GeoStudio does not natively have a command line tool or similar for this! Have you or anyone tried batch analyses with your code?

Thanks,
Nicki

**Comment by MoiseRousseau** [comment]
Hi Nicki,
PyGeoStudio can acheive that. Here is an example from the documentation: https://pygeostudio.readthedocs.io/en/stable/gallery_examples/parametric/rapid_drawdown_Ksat.html#sphx-glr-gallery-examples-parametric-rapid-drawdown-ksat-py
Which parameter do you want to vary ? It may not be implemented for instance.
Moise

**Reply by nshobert** [reply]
Thanks, that example was helpful. That is basically the pattern I want to use.  For now, my focus will be varying soil material parameters (unit weight, phi, etc.) Is that implemented yet?

**Comment by MoiseRousseau** [comment]
Are you using slope or sigma ? If yes, you will deal with untested workflow, as my primary focus was seep. If you have time to test, we can keep informed each other to implement that feature.
